### ISSUE TITLE
Support to add tensorboard file, or file location

### ISSUE THREAD

**Issue by nikhileshva** [ModelChimp/modelchimp#7]
Current scenario: 
I can save the hyperparameters in ModelChimp. However, there are some additional graphs, apart from accuracy and loss, that I plot in Tensorboard. 
Then I have to save the folder name in the list of hyperparameters. 
Works for now. 

But, it would be good if I can save the tensorboard file in ModelChimp.

**Comment by samzer** [comment]
Yes. We are working on this feature. You'll be able to save any kind of files or folder.  This should be done by mid-March.
